tests/patch_time.py

```diff
@@ -20,10 +16,21 @@ def _monotonic() -> float:
     return time.monotonic()
 
 
-# Replace partial functions which are not found by freezegun
+# Before importing anything else, replace partial dt_util.utcnow
+# with a regular function which can be found by freezegun
+from homeassistant import util  # noqa: E402
```

Comment:
This import can be at the top

Reply:
Yes, but let's not try to be smart, let's be very explicit about what we're doing and why